test/skate/settings/test_group_test.exs

```diff
@@ -97,6 +119,37 @@ defmodule Skate.Settings.TestGroupTest do
       assert user2 in users_update_2
       assert user3 in users_update_2
     end
+
+    test "can set override to enabled" do
+      {:ok, test_group} = TestGroup.create("group name")
+
+      new_test_group = TestGroup.update(%{test_group | override: :enabled})
+
+      assert new_test_group.override == :enabled
+    end
+
+    test "override defaults to :none" do
+      {:ok, test_group} = TestGroup.create("group name")
+
+      assert test_group.override == :none
+    end
+
+    test "override shows as :none if it is nil in the database" do
+      {:ok, test_group} = TestGroup.create("group name")
+
+      new_test_group = TestGroup.update(%{test_group | override: nil})
+
+      assert new_test_group.override == :none
+    end
+
+    @tag :skip
```

Comment:
I could be persuaded to remove this test for now, since there isn't really a good way of denoting that an update failed. I was planning on eventually switching `TestGroup.update/1` to also return `{:ok, tg} | {:error, errors}`, like I did for `TestGroup.create/1` a while ago.

Reply:
I think so, but I didn't bother investigating it too thoroughly, since there isn't _really_ a way of triggering that behavior.